src/unstract/sdk/adapter.py

```diff
@@ -65,7 +65,8 @@ def _get_adapter_configuration(
             # TODO: Print config after redacting sensitive information
             self.tool.stream_log(
                 "Successfully retrieved config "
-                f"for adapter instance {adapter_instance_id}"
+                f"for adapter instance {adapter_instance_id}",
```

Comment:
Will it help to log adapter model name also as DEBUG? I think you had suggested this long back...

Reply:
Added some context here, later we can include the entire metadata after redacting sensitive information. This will allow us to debug issues with full context and we needn't go back to the user with questions